services/chronograf/values-squash-sandbox.yaml

```diff
@@ -0,0 +1,88 @@
+chronograf:
+  ## Image Settings
+  ##
+  image:
+    repository: "chronograf"
+    tag: 1.8.6
+    pullPolicy: IfNotPresent
```

Comment:
Does upstream release a new version of the Helm chart for each version of the software? If so, it may be easier to use their defaults to pick up new versions. This way, we have to remember to upgrade the underlying application as well as the Helm chart, and Renovate may warn us when there's a new application available. (It can sometimes figure this out but not always.)

Reply:
Unfortunately they don't, the InfluxData Helm charts are behind.  Right now they have  Chronograf 1.8.0, InfluxDB 1.8.0-alpine and Kapacitor 1.5.4-alpine.